Backend/nix/postgres-with-replica.nix

```diff
@@ -0,0 +1,71 @@
+{ config, pkgs, lib, ... }:
+let
+  inherit (lib) types;
+in
+{
+  options = {
+    services.postgres-with-replica = lib.mkOption {
+      type = types.attrsOf (types.submodule {
```

Comment:
Perhaps add a `description` here describing what this submodule (postgres-with-replica) does in effect?